File: CursoPOOUber/Java/UberX.java
package CursoPOOUber.Java;

//Definition of the "UberX" class, which inherits form the "Car" class.
class UberX extends Car {

    //Attributes
    String brand;
    String model;
    
    public UberX(String license, Account driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
        setPassengers(4);
    }

    void printUberxData(){
        System.out.println("Car Brand: " + brand + " UberX Model: " + model);
    }
}

File: CursoPOOUber/JS/UberX.js
//Definition of the "UberX" class, which inherits form the class "Car".

class UberX extends Car {
    constructor(license, driver, brand, model) {
        super(license, driver)
        this.brand = brand;
        this.model = model;
    }

    printDataCar() {
        super.printDataCar();
        console.log("Car Brand: " + this.brand + " UberX Model: " + this.model);
    }
}